proto/bragi/common.proto

```diff
@@ -59,6 +59,9 @@ message Match {
   uint32 away_score = 7;
   MatchStatusType match_status = 8;
   google.protobuf.Timestamp plannet_start = 9;
+  string tournament_name = 10;
+  string home_team_name = 11;
```

Comment:
There is already `home_team_urn` now name added. I feel like it should be replaced with `Team` object. Similar for tournament.

Reply:
I agree. I will add the object but still have `home_team_urn` so that we don't introduce a breaking change. But I will deprecate the field

Reply:
ok

Reply:
Refactored